.github/workflows/ci-linux-on-main-pr.yml

```diff
@@ -1,21 +1,19 @@
 name: "Linux CI"
 
 on:
-  workflow_dispatch:
   push:
     branches:
-    - main
+      - main
```

Comment:
I could be wrong about this, but I thought the original alignment here was correct, where the `-` lines up with the overarching label. Like this:
```yaml
  branches:
  - main
```
instead of:
```yaml
  branches:
    - main
```

Reply:
This is autocorrected by the YAML style enforcer, so my assumption is that it is correct (plus GH normally complains when you give it invalid YAML).